test/localization/int.spec.ts

```diff
@@ -1869,7 +1869,9 @@ describe('Localization', () => {
       })
     })
 
-    describe('nested localized field sanitization', () => {
+    // Disabled, as nested localized fields do no longer have their localized property stripped in
+    // this monorepo, as this is handled at runtime.
+    describe.skip('nested localized field sanitization', () => {
```

Comment:
Either delete the test as it is no longer doing anything or change the assertions to `toBeDefined()` and keep the test.

Reply:
adjusted it